modules/persistent-storage-csi-azure-file-nfs.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent_storage-csi-azure-file.adoc
+//
+
+:_content-type: CONCEPT
+[id="persistent-storage-csi-azure-file-nfs_{context}"]
+= NFS support
+
+{product-title} 4.14, and later, supports Azure File Container Storage Interface (CSI) Driver Operator with  Network File System (NFS) with the following caveats:
+
+* Azure File with NFS applies a FS group policy at the driver level and pods cannot override it.
+
+* The Azure File CSI Operator does not automatically create a storage class for NFS. You need to create it manually.
```

Comment:
We need to add a snippet of the storage class as per our testing.
cc @gnufied @bertinatto @duanwei33

Reply:
Yeah so something like:

```
apiVersion: storage.k8s.io/v1
kind: StorageClass
metadata:
  name: azurefile-csi-nfs
provisioner: file.csi.azure.com
parameters:
  protocol: nfs
  skuName: Premium_LRS  # available values: Premium_LRS, Premium_ZRS
mountOptions:
  - nconnect=4
 ```

Reply:
Done.